numpyro/util.py

```diff
@@ -69,7 +68,7 @@ def fori_loop(lower, upper, body_fun, init_val):
             val = body_fun(i, val)
         return val
     else:
-        return jit(lax.fori_loop, static_argnums=(2,))(lower, upper, body_fun, init_val)
+        return lax.fori_loop(lower, upper, body_fun, init_val)
```

Comment:
Is the issue of slow compilation resolved with the latest release?

Reply:
Yes, it is fixed the last minute. :)